src/index.js

```diff
@@ -98,7 +98,11 @@ async function cli () {
   // if an input file is defined, it should take precedence, otherwise, input is
   // coming from stdin and just name the file out.svg, if it hasn't been
   // specified with the '-o' option
-    output = input ? (input + '.svg') : 'out.svg'
+    if(outputFormat) {
```

Comment:
I think you need to do the following to get the linter happy.

It should be automatically fixed by `yarn run lint-fix` too.

```suggestion
    if (outputFormat) {
```